include/psa/crypto_extra.h

```diff
@@ -121,6 +121,14 @@ psa_status_t mbedtls_psa_inject_entropy(const unsigned char *seed,
                                         size_t seed_size);
 
 
+/* The Number of key slots (plus one because 0 is not used).
+ * The value is a compile-time constant for now, for simplicity. */
+#if defined(MBEDTLS_PSA_KEY_SLOT_COUNT)
```

Comment:
```suggestion
#if !defined(MBEDTLS_PSA_KEY_SLOT_COUNT)
```

Reply:
I was told the preffernce was for positive ifdef checks as opposed to negative ones.
I agree that the negative route is shorter. I don't mind changing if this doesn't present a problem.

Reply:
great, thanks @gilles-peskine-arm,  will fix this to the simpler version

Reply:
fixed